src/bundle/Resources/public/js/scripts/core/tag.view.select.js

```diff
@@ -88,11 +88,12 @@
         }
 
         removeItems(items) {
+            const stringifiedItems = items.map(String);
```

Comment:
I think it would be cleaner to perform all castings only in one method of the component, and if want to choose one I think it has to be `addItems`; in such case, casting would be in line:
`deleteButton.addEventListener('click', () => this.removeItem(String(id)), false);`
with this one cast, we do not need more castings here and the code should be cleaner.
WDYT @ibexa/javascript-dev  ?